README.md

```diff
@@ -1,15 +1,13 @@
-# Adobe Experience Platform Media For Edge Network Extension
-
-[![Cocoapods](https://img.shields.io/cocoapods/v/AEPEdgeMedia.svg?color=orange&label=AEPEdgeMedia&logo=apple&logoColor=white)](https://cocoapods.org/pods/AEPEdgeMedia)
-
-[![SPM](https://img.shields.io/badge/SPM-Supported-orange.svg?logo=apple&logoColor=white)](https://swift.org/package-manager/)
-[![CircleCI](https://img.shields.io/circleci/project/github/adobe/aepsdk-edgemedia-ios/main.svg?logo=circleci)](https://circleci.com/gh/adobe/workflows/aepsdk-edgemedia-ios)
-[![Code Coverage](https://img.shields.io/codecov/c/github/adobe/aepsdk-edgemedia-ios/main.svg?logo=codecov)](https://codecov.io/gh/adobe/aepsdk-edgemedia-ios/branch/main)
+# Adobe Experience Platform Streaming Media For Edge Network Extension
 
+[![CocoaPods](https://img.shields.io/github/v/release/adobe/aepsdk-edgemedia-ios?label=CocoaPods&logo=apple&logoColor=white&color=orange)](https://cocoapods.org/pods/AEPEdgeMedia)
+[![SPM](https://img.shields.io/github/v/release/adobe/aepsdk-edgemedia-ios?label=SPM&logo=apple&logoColor=white&color=orange)](https://github.com/adobe/aepsdk-edgemedia-ios/releases)
+[![CircleCI](https://img.shields.io/circleci/project/github/adobe/aepsdk-edgemedia-ios/main.svg?label=Build&logo=circleci)](https://circleci.com/gh/adobe/workflows/aepsdk-edgemedia-ios)
+[![Code Coverage](https://img.shields.io/codecov/c/github/adobe/aepsdk-edgemedia-ios/main.svg?label=Coverage&logo=codecov)](https://codecov.io/gh/adobe/aepsdk-edgemedia-ios/branch/main)
 
 ## About this project
 
-The AEP Media Analytics for Edge Network mobile extension provides clients with robust measurement for audio, video and advertisements when using the [Adobe Experience Platform Mobile SDK](https://developer.adobe.com/client-sdks) and the Edge Network extension.
+The Adobe Streaming Media on Edge Network extension sends data about audio and video consumption on your streaming applications to the Adobe Experience Platform Edge Network. This enables capabilities for measurement, analysis, and activation with media data across the Adobe Experience Cloud solutions when using the [Adobe Experience Platform Mobile SDK](https://developer.adobe.com/client-sdks) and the Edge Network extension.
```

Comment:
Media for Edge

Reply:
agree.  The Adobe Streaming Media for Edge Network extension sends data...

Reply:
Sorry, my mistake. I cut and pasted this from the email 😄